frontend/projects/abtesting/src/app/features/dashboard/home/components/modal/new-experiment/new-experiment.component.ts

```diff
@@ -49,6 +51,19 @@ export class NewExperimentComponent implements OnInit {
           ...this.newExperimentData,
           ...formData
         };
+
+        if (!this.currentContext && this.experimentInfo) {
+          this.currentContext = this.experimentInfo.context[0];
+        }
+
+        this.contextChanged = this.currentContext ? (this.currentContext !== this.newExperimentData.context[0]) : false;
```

Comment:
Update variable name to isContextChanged

Reply:
code updated